cdc/owner/ddl_sink.go

```diff
@@ -54,6 +54,7 @@ type DDLSink interface {
 	// the DDL event will be sent to another goroutine and execute to downstream
 	// the caller of this function can call again and again until a true returned
 	emitDDLEvent(ctx context.Context, ddl *model.DDLEvent) (bool, error)
+	emitBootstrapEvent(ctx context.Context, ddl *model.DDLEvent) error
```

Comment:
can we use the `emitDDLEvent` directly ?

Reply:
I have tried, but it will bring some complexity into the `emitDDLEvent` interface.

Reply:
It seems that `emitDDLEvent` will not be called until `emitBootstrapEvent` completes, and the only difference between them is whether or not the caller is blocked, so perhaps this behavior could be controlled by adding an argument to `emitDDLEvent`?

Reply:
I also think it is better to separate this two function.